### PR TITLE
chore: change mau count api in the notification sender

### PR DESCRIPTION
**Pull request by cre8ivejp**
I'm changing the API to get the count from MySQL instead of Druid.

### Things done

- add GetMAUCount api to the event counter service
- change mau count api in the notification sender

### Log

```
{"severity":"INFO","eventTime":1672117994.888366,"logger":"get-mau-count.get","caller":"get-mau-count/command.go:73","message":"Get the count successfully.","serviceContext":{"service":"get-mau-count.get","version":"-"},"yearMonth":"202212","userCount":617,"eventCount":1831}
```

### CODE REVIEW

File: pkg/notification/sender/informer/batch/job/mau_count_watcher_test.go [hunk 1]
```diff
@@ -77,13 +68,10 @@ func TestCreateMAUNotification(t *testing.T) {
 					&environmentproto.ListProjectsResponse{
 						Projects: []*environmentproto.Project{{Id: "pj0"}},
 						Cursor:   "cursor",
-					}, nil).Times(1)
```
Comment (cre8ivejp) [inline pkg/notification/sender/informer/batch/job/mau_count_watcher_test.go:80]: By default, the Times is set to one, so this is unnecessary.

File: pkg/eventcounter/api/api.go [hunk 2]
```diff
@@ -824,6 +826,43 @@ func (s *eventCounterService) GetUserCountV2(
...
+	ctx context.Context,
+	req *ecproto.GetMAUCountRequest,
+) (*ecproto.GetMAUCountResponse, error) {
+	localizer := locale.NewLocalizer(locale.NewLocale(locale.JaJP))
+	_, err := s.checkRole(ctx, accountproto.Account_VIEWER, req.EnvironmentNamespace, localizer)
+	if err != nil {
+		return nil, err
+	}
+	if req.YearMonth == "" {
+		return nil, localizedError(statusMAUYearMonthRequired, locale.JaJP)
```
Comment (ono-max) [inline pkg/eventcounter/api/api.go:839]: Can you change `localizedError` to the following implementation?

https://github.com/bucketeer-io/bucketeer/blob/main/pkg/feature/api/feature.go#L64

File: pkg/notification/sender/informer/batch/job/mau_count_watcher_test.go [hunk 3]
```diff
@@ -208,6 +197,16 @@ func TestCreateMAUNotification(t *testing.T) {
 	}
 }
 
+func TestGeteYearLastMonth(t *testing.T) {
```
Comment (ono-max) [inline pkg/notification/sender/informer/batch/job/mau_count_watcher_test.go:200]: nit:
```suggestion
func TestGetYearLastMonth(t *testing.T) {
```

**Comment by cre8ivejp** [comment]
@ono-max, I pushed the fixes. Thank you!

File: pkg/eventcounter/api/error.go [hunk 4]
```diff
@@ -124,6 +132,8 @@ func localizedError(s *gstatus.Status, loc string) error {
 		return errFeatureIDRequiredJaJP
 	case statusExperimentIDRequired:
 		return errExperimentIDRequiredJaJP
+	case statusMAUYearMonthRequired:
```
Comment (ono-max) [inline pkg/eventcounter/api/error.go:135]: Since, you don't use localizedError, we can remove this line 👍

File: pkg/eventcounter/api/error.go [hunk 5]
```diff
@@ -57,6 +58,13 @@ var (
 			Message: "experiment idは必須です",
 		},
 	)
+	errMAUYearMonthRequiredJaJP = status.MustWithDetails(
```
Comment (ono-max) [inline pkg/eventcounter/api/error.go:61]: Same as https://github.com/bucketeer-io/bucketeer/pull/136/files#r1057493768

**Comment by cre8ivejp** [comment]
@ono-max, done! Thanks.

**Review by ono-max — APPROVED** [review]
Thank you 👍

**Pull request merged**